sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3180,7 +3180,40 @@ pi_result piDeviceGetInfo(pi_device Device, pi_device_info ParamName,
     }
     return ReturnValue(FreeMemory);
   }
+  case PI_EXT_INTEL_DEVICE_INFO_MEMORY_CLOCK_RATE: {
+    // If there are no any memory modules then return 0.
```

Comment:
Let's document this behavior in pi.h

Reply:
And in the .md too, and also explain what is returned when a device has multiple memories

Reply:
Updated pi.h and document.